modules/rosa-adding-tags-cli.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-adding-tags-cli{context}"]
+= Adding tags to a machine pool using the ROSA CLI
+
+You can add tags to a machine pool for your {product-title} cluster by using the ROSA CLI.
+
+.Prerequisites
+
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
+* You logged in to your Red Hat account by using the `rosa` CLI.
+* You created a {product-title} (ROSA) cluster.
+
+.Procedure
+
+. List the machine pools in the cluster by running the following command:
++
+[source,terminal]
+----
+$ rosa list machinepools --cluster=<cluster_name>
+----
++
+.Example output
++
+[source,terminal]
+----
+ID           AUTOSCALING  REPLICAS  INSTANCE TYPE  LABELS  TAGS  TAINTS    AVAILABILITY ZONES    SPOT INSTANCES     DISK SIZE   SG IDs
+Default      No           2         m5.xlarge                              us-east-1a            N/A                300 GiB     sg-0e375ff0ec4a6cfa2
+db-nodes-mp  No           2         m5.xlarge                              us-east-1a            No                 300 GiB     sg-0e375ff0ec4a6cfa2
+----
+
+. Add or update the tags for a machine pool:
+
+* To add or update tags for a machine pool, run the following command:
++
+[source,terminal]
+----
+$ rosa edit machinepool --cluster=<cluster_name> \
+                        --replicas=<replica_count> \
+                        --tags=<key>=<value>:<effect>,<key>=<value>:<effect> \ <1>
```

Comment:
I think these are taints instead of tags :thinking: 

The aws tags can't be updated after creation